README.md

```diff
@@ -53,7 +53,7 @@
 - A: Unfortunately some sections of Prism Launcher cannot be themed due to Qt using them.
 ---
 - Q: "Can I change the color?"
-- A: If you would like to change the accent/highlighted color, you can do this by modifying the `theme.json` file. Then editing the `"Highlight"` value to any color you prefer. You can also do this to other values if you would like.
+- A: If you would like to change any colors, you can do this by modifying the `theme.json` file. Then editing the the values to any color you prefer.
```

Comment:
Not sure if any of this section should be here seeing as we already pre-generate all flavor + accent combinations? What other colors would need to be changed?

Reply:
Good point. I have removed that question from the FAQ.